libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
Worth noting that for synthetic tests we already have a default results dir (https://github.com/ARM-software/lisa/blob/master/libs/utils/env.py#L259), would be good to keep them aligned.